From: Jiri Slaby <jslaby@suse.cz>
Date: Mon, 18 Jan 2016 13:19:54 +0100
Subject: kgr: usb/storage: do not emit thread awakened
Patch-mainline: not yet, kgraft
References: bnc#899908

It is unwanted to emit thread awakened when woken only for kgraft.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/usb/storage/usb.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/storage/usb.c
+++ b/drivers/usb/storage/usb.c
@@ -315,13 +315,13 @@ static int usb_stor_control_thread(void
 		if (to < 0)
 			break;
 
-		usb_stor_dbg(us, "*** thread awakened\n");
-
 		if (!to) {
 			kgr_task_safe(current);
 			continue;
 		}
 
+		usb_stor_dbg(us, "*** thread awakened\n");
+
 		/* lock the device pointers */
 		mutex_lock(&(us->dev_mutex));
 
